src/pages/AddLiquidity/index.tsx

```diff
@@ -641,7 +665,8 @@ export default function AddLiquidity({
                     />
                   </AutoColumn>
                 </DynamicSection>
-              </div>
+                {!hasExistingPosition && <LayerTwoAddLiquidityPromotion suggestL2={showL2Suggestion} />}
```

Comment:
it looks like `LayerTwoAddLiquidityPromotion` is doing its own conditional render internally. Would it make sense to combine the two checks and put them in one place?

Reply:
Done. I've removed `suggestL2` as it's solely used by the 5 ETH condition. Now the prompt shows up when the user enters the Add Liquidity page, even before Currency B is chosen.